diff --git a/node_modules/react-scripts/scripts/start.js b/node_modules/react-scripts/scripts/start.js
index 35bce79..72255cb 100644
--- a/node_modules/react-scripts/scripts/start.js
+++ b/node_modules/react-scripts/scripts/start.js
@@ -94,7 +94,7 @@ checkBrowsers(paths.appPath, isInteractive)
     const config = configFactory('development');
     const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
     const appName = require(paths.appPackageJson).name;
-    const useTypeScript = fs.existsSync(paths.appTsConfig);
+    const useTypeScript = fs.existsSync(paths.appTsConfig) && !process.argv.includes('--skip');
     const urls = prepareUrls(protocol, HOST, port);
     const devSocket = {
       warnings: warnings =>
