diff --git a/node_modules/@emotion/core/types/index.d.ts b/node_modules/@emotion/core/types/index.d.ts
index a4ade59..fc70bea 100644
--- a/node_modules/@emotion/core/types/index.d.ts
+++ b/node_modules/@emotion/core/types/index.d.ts
@@ -81,21 +81,3 @@ export function ClassNames<Theme = any>(
   props: ClassNamesProps<Theme>
 ): ReactElement<any>
 
-declare module 'react' {
-  interface DOMAttributes<T> {
-    css?: InterpolationWithTheme<any>
-  }
-}
-
-declare global {
-  namespace JSX {
-    /**
-     * Do we need to modify `LibraryManagedAttributes` too,
-     * to make `className` props optional when `css` props is specified?
-     */
-
-    interface IntrinsicAttributes {
-      css?: InterpolationWithTheme<any>
-    }
-  }
-}
